web/web.go

```diff
@@ -247,8 +247,8 @@ func serveDebug(w http.ResponseWriter, req *http.Request) {
 	subpath := route.Param(ctx, "subpath")
 
 	// Based off paths from init() in golang.org/src/net/http/pprof/pprof.go
-	if subpath == "/pprof/" {
-		pprof.Index(w, req)
+	if subpath == "/pprof" {
+		http.Redirect(w, req, req.URL.Path+"/", http.StatusMovedPermanently)
```

Comment:
The redirect also attempts to match the behaviour of the ServeMux: https://golang.org/pkg/net/http/#ServeMux